From 021330d1a1df8ca816e8d4e1bea26112397052db Mon Sep 17 00:00:00 2001
From: Jesse Pretorius <jesse.pretorius@rackspace.co.uk>
Date: Fri, 23 Jun 2017 13:21:47 +0100
Subject: [PATCH] Remove ssh key generation

Some tasks can't be changed upstream and have to be fixed here.
---
 tasks/keystone_pre_install.yml | 1 -
 1 file changed, 1 deletion(-)

diff --git a/tasks/keystone_pre_install.yml b/tasks/keystone_pre_install.yml
index 8a8a85c..c348d46 100644
--- a/tasks/keystone_pre_install.yml
+++ b/tasks/keystone_pre_install.yml
@@ -46,7 +46,6 @@
     system: "yes"
     createhome: "yes"
     home: "{{ keystone_system_user_home }}"
-    generate_ssh_key: "yes"

 - name: Create keystone dir
   file:
--
2.11.0 (Apple Git-81)

